news/2017-10-09-halloween-fanart-contest.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Halloween 2017 Fanart Contest"
+date: 2017-10-09 06:00:00 +0000
+---
+
+Spooks, chills and thrills abound! Halloween is drawing close again, and you know what that means - a fanart contest! Two hectic weeks to pen any of the osu! mascots (or community mascots) in a delightfully spooky Halloween setting, for a chance at getting your work added to the game as a seasonal background!
+
+[![](https://assets.ppy.sh/contests/56/header.jpg)](https://osu.ppy.sh/community/contests/56)
+
+Halloween's quickly becoming something of an _osu!_ staple as far as seasonal events go, what with the [incredible showing](https://osu.ppy.sh/home/news/2016-10-29-halloween-cometh) from last year's spookiness.
+
+We're aiming to match the glory of last year's showing, and to kick off the preparations, we're hosting another **Halloween-themed fanart contest, including all of the osu! community mascots!**
+
+Like last year, we'll be taking a top numbered selection of the community's favourite entries (as determined via vote) and slapping them on the osu! in-game main menu as seasonal backgrounds for everybody to admire. Each winner will also receive **2 months of osu!supporter** for either themselves or a friend, or themselves and a friend, or two friends, or.. you get the idea.
+
+The intrepid artists among us have **14 days** from the date of this post to put pen to paper and render our lovable mascots in all their (terrifying) glory - so long as it is Halloween themed!
+
+That's two weeks, for those who aren't numerically inclined. Two weeks isn't a whole lot of time either, so make sure to get cracking! We'll be holding the vote pretty much as soon as the contest closes.
+
+(It's also [Mocha's](https://assets.ppy.sh/media/mocha/Mocha_Documentation.pdf) first Halloween ever. Make of that what you will, but don't leave her out of the festivities!)
+
+Check out the rules below before you start:
+
+Rules
+-----
+ 
+* **Artists should use the custom templates provided: [PSD template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.psd?2017) / [PNG template](https://assets.ppy.sh/events/fanart/templates/osu%21%20main%20menu%202732x1536.png?2017).** By use the templates we DON'T mean "leave the template visible in the artwork" - that just looks awful. The template shows what the in-game osu! cookie will hide. Make sure to **NOT** leave the template visible in your submission. Use it as a guide to figure out how to compose your piece, and that's it!
```

Comment:
`By "use the templates," we DON'T mean`
put quotes around `use the templates` and a comma after `templates`

Reply:
good catch, fixing